tck/README.adoc

```diff
@@ -16,31 +16,53 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-
 = MicroProfile Reactive Messaging TCK
 
-This project contains the TCK for MicroProfile Reactive Messaging.
+This project contains the TCK for Reactive Messaging
+
+
+== Checking an implementation against the TCK
 
-== Running the TCK against an implementation
+In your implementation, create a new module depending on the TCK artifact:
 
-The TCK uses http://arquillian.org/[Arquillian], so an implementation needs Arquillian container support to run. In addition, the implementation must also implement the TCK's SPI in order to allow the TCK to interact and make assertions on messages sent and received from topics under test.
+[source,xml]
+----
+<dependency>
+    <groupId>org.eclipse.microprofile.reactive.messaging</groupId>
+    <artifactId>microprofile-reactive-messaging-core-tck</artifactId>
+    <version>${microprofile-reactive-messaging.version}</version>
+</dependency>
+----
 
-=== `TckContainer`
+This project will likely also depends on your implementation.
```

Comment:
Suggest "Your test project that runs the TCK is also very likely to depend on your implementation project."